src/main/kotlin/onm/human/Human.kt

```diff
@@ -0,0 +1,60 @@
+package onm.human
+
+import onm.house.devices.AbstractDevice
+import onm.house.devices.Dryer
+import onm.house.devices.Fridge
+import onm.things.Equipment
+import onm.things.Food
+import onm.things.FoodType
+import onm.units.HumanControlUnit
+import kotlin.concurrent.thread
+
+class Human(val ability: HumanAbility, val name:String) {
+    var available: Boolean = true
+
+    var controlUnit: HumanControlUnit? = null
```

Comment:
Why is this nullable? Every human must have control unit. We shall then have something like human creator/factory which will create Humans and add them control unit e.g., to constructor. And House builder would use this creator/factory